src/main/java/com/google/sps/data/Marker.java

```diff
@@ -32,15 +31,13 @@
   public Marker(
       double newLat,
       double newLng,
-      Long newId,
```

Comment:
I think this might be required again in future for supporting editing of existing routes.